go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/containerd/nerdctl
 go 1.16
```

Comment:
Probably you can omit all changes in this file

Reply:
done.